odw/services/data-warehouse/app/ingest/models.py

```diff
@@ -147,6 +147,7 @@ class UnstructuredData(BaseModel):
 class Medical(BaseModel):
     id: Key[str]
     patient_name: str
+    patient_age: str
```

Comment:
Could this be an int?

Reply:
It could but I hesitated making that change since there are many other places where we should make those changes